### ISSUE TITLE
`tsdoc` syntax issue: `tsdoc-inline-tag-missing-braces: The TSDoc tag "@link" is an inline tag; it must be enclosed in "{ }" braces`

### ISSUE THREAD

**Issue by komali2** [kubb-labs/kubb#1421]
### What version of `kubb` is running?

3.0.5

### What kind of platform do you use?

Linux

### How does your `kubb.config.ts` config look like

```typescript
import { defineConfig } from "@kubb/core";
import { pluginTs } from "@kubb/plugin-ts";
import { pluginZod } from "@kubb/plugin-zod";
import { pluginReactQuery } from "@kubb/plugin-react-query";
import { pluginOas } from "@kubb/plugin-oas";
import { pluginClient } from "@kubb/plugin-client";

export default defineConfig({
  root: ".",
  input: {
    path: "./apps/backend/openapi.json",
  },
  output: {
    path: "./apps/webapp/src/gen",
    clean: true,
  },
  hooks: {
    done: ["npm run lintfix", "npm run format"],
  },
  plugins: [
    pluginOas(),
    pluginTs({
      output: {
        path: "models",
      },
      group: {
        type: "tag",
        name: ({ group }) => `${group}Controller`,
      },
      enumType: "literal",
    }),
    pluginClient({
      output: {
        path: ".",
      },
      importPath: "./kubb-client.ts",
    }),
    pluginZod({
      output: {
        path: "./zod",
      },
      group: {
        type: "tag",
        name: ({ group }) => `${group}Schemas`,
      },
      typed: true,
      dateType: "stringOffset",
      unknownType: "unknown",
      importPath: "zod",
    }),
    pluginReactQuery({
      output: {
        path: "./hooks",
      },
      group: {
        type: "tag",
        name: ({ group }) => `${group}Hooks`,
      },
      client: {
        dataReturnType: "full",
      },
      mutation: {
        methods: ["post", "put", "delete"],
      },
      infinite: {
        queryParam: "page",
        initialPageParam: 0,
        cursorParam: undefined,
      },
      query: {
        methods: ["get"],
        importPath: "@tanstack/react-query",
      },
      parser: "zod",
      pathParamsType: "object",
    }),
  ],
});

```

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

```JSON
{
  "name": "my-turborepo",
  "private": true,
  "scripts": {
    "build": "turbo build",
    "dev": "turbo dev",
    "lint": "turbo lint -- --quiet",
    "lintfix": "turbo lint -- --fix",
    "format": "prettier  --write \"**/*.{ts,tsx,md}\" \"!**/docker/.volumes/postgres/**\"",
    "formatcheck": "prettier --check \"**/*.{ts,tsx,md}\" \"!**/docker/.volumes/postgres/**\"",
    "generate": "kubb generate"
  },
  "devDependencies": {
    "@kubb/cli": "^3.0.5",
    "@kubb/core": "^3.0.5",
    "@kubb/plugin-client": "^3.0.5",
    "@kubb/plugin-oas": "^3.0.5",
    "@kubb/plugin-react-query": "^3.0.5",
    "@kubb/plugin-ts": "^3.0.5",
    "@kubb/plugin-zod": "^3.0.5",
    "@types/plotly.js": "^2.33.4",
    "postcss": "^8.4.47",
    "postcss-preset-mantine": "^1.17.0",
    "postcss-simple-vars": "^7.0.1",
    "prettier": "^3.2.5",
    "turbo": "^2.1.2",
    "typescript": "^5.4.5",
    "zod": "^3.23.8"
  },
  "engines": {
    "node": ">=18"
  },
  "packageManager": "npm@10.5.0",
  "workspaces": [
    "apps/*",
    "packages/*"
  ],
  "dependencies": {
    "@prisma/client": "^5.18.0",
    "@radix-ui/react-tooltip": "^1.1.3"
  }
}
```

### What steps can reproduce the bug?

Running `kubb` with a config that will generate tanstack query hooks will also generate `link` tsdoc comments that look like:

```
 /**
 * @summary Find steps
 * @link /steps
 */
```

Apparently, this is syntactically incorrect for `@link` type tags. I get the following tslint error:

```
│ 7:4  Error: tsdoc-inline-tag-missing-braces: The TSDoc tag "@link" is an inline tag; it must be enclosed in "{ }" braces  tsdo
```

I believe they're supposed to look like this:

```
 * {@link /steps}
 ```
 
 I tried re-defining the `link` tag type in my `tsdoc.json` but that's not allowed and doesn't fix the issue:
 
 ```
 {
    "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
    "tagDefinitions": [
        {
            "tagName": "@description",
            "syntaxKind": "block"
        },
        {
            "tagName": "@default",
            "syntaxKind": "block"
        },
        {
            "tagName": "@type",
            "syntaxKind": "block"
        },
        {
            "tagName": "@summary",
            "syntaxKind": "block"
        },
        {
            "tagName": "@link",
            "syntaxKind": "block"
        }
    ],
    "supportForTags": {
        "@description": true,
        "@default": true,
        "@type": true,
        "@summary": true,
        "@link": true
    }
}

```

### How often does this bug happen?

Every time

**Comment by stijnvanhulle** [comment]
Fixed with v`3.0.11`

**Comment by komali2** [comment]
confirmed fixed!
